src/components/InvoiceList/index.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react"
+import InvoiceSummary from "../InvoiceSummary"
+
+export default function InvoiceList() {
+
+    const [invoices, setInvoices] = useState([])
+    useEffect(() => {
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices').then(res => res.json()).then(
```

Comment:
Can you put the then blocks onto different lines to be consistent with the request in header